src/components/v2/appDetails/SyncError.component.tsx

```diff
@@ -1,28 +1,60 @@
-import React, { useState } from 'react';
+import React, { useState } from 'react'
 
-import { ReactComponent as DropDownIcon } from '../../../assets/icons/ic-chevron-down.svg';
-import { ReactComponent as AlertTriangle } from '../../../assets/icons/ic-alert-triangle.svg';
-import { not } from '../../common';
+import { ReactComponent as DropDownIcon } from '../../../assets/icons/ic-chevron-down.svg'
+import { ReactComponent as AlertTriangle } from '../../../assets/icons/ic-alert-triangle.svg'
+import { not } from '../../common'
+import IndexStore from './index.store'
+import { ConditionContainer } from '../../CIPipelineN/ConditionContainer'
+
+const SyncErrorComponent: React.FC<{ appStreamData; showApplicationDetailedModal? }> = ({
+    appStreamData,
+    showApplicationDetailedModal,
+}) => {
+    const [collapsed, toggleCollapsed] = useState<boolean>(true)
+    const appDetails = IndexStore.getAppDetails()
+    const conditions = appStreamData?.result?.application?.status?.conditions || []
+
+    let isImagePullBackOff
+    for (let index = 0; index < appDetails.resourceTree.nodes.length; index++) {
+        const node = appDetails.resourceTree.nodes[index]
+        if (node.info?.length) {
+            for (let index = 0; index < node.info.length; index++) {
+                const info = node.info[index]
+                if (info.value.toLowerCase() === 'errimagepull' || info.value.toLowerCase() === 'imagepullbackoff') {
+                    isImagePullBackOff = true
+                    break
+                }
+            }
+            if (isImagePullBackOff) break
+        }
+    }
+
+    if (conditions.length === 0 && !isImagePullBackOff) return null
+
+    const renderConditionErrorMessage = () => {
+        return !appDetails?.ipsAccessProvided ? (
```

Comment:
We can check appDetails on the top and remove all other optional checks from aapDetails